modules/candidate_parameters/php/NDB_Form_candidate_parameters.class.inc

```diff
@@ -148,9 +148,11 @@ class NDB_Form_Candidate_Parameters extends NDB_Form
      */
     function getJSDependencies()
     {
-        $factory = NDB_Factory::singleton();
-        $baseURL = $factory->settings()->getBaseURL();
-        $deps    = parent::getJSDependencies();
+        $factory    = NDB_Factory::singleton();
+        $baseURL    = $factory->settings()->getBaseURL();
+        $deps       = parent::getJSDependencies();
+        $useConsent = $this->tpl_data['useConsent'];
```

Comment:
This should probably go into the LorisHelper global object config() instead of a URL?

See here: https://github.com/aces/Loris/blob/17.1-dev/smarty/templates/main.tpl#L14